### PR TITLE
Added ip_to_ptr function

### PR DESCRIPTION
**Pull request by qduk**

### CODE REVIEW

File: netutils/ip.py [hunk 1]
```diff
@@ -387,3 +387,23 @@ def get_usable_range(ip_network):
         lower_bound = str(net[1])
         upper_bound = str(net[-2])
     return f"{lower_bound} - {upper_bound}"
+
+
+def ip_to_ptr(ip_address):
+    """Generates a ptr record from a given IPv4 address.
```
Comment (itdependsnetworks) [inline netutils/ip.py:393]: Update to inclued ipv6.

Comment (qduk) [reply netutils/ip.py]: Updated

File: netutils/ip.py [hunk 2]
```diff
@@ -387,3 +387,23 @@ def get_usable_range(ip_network):
         lower_bound = str(net[1])
         upper_bound = str(net[-2])
     return f"{lower_bound} - {upper_bound}"
+
+
+def ip_to_ptr(ip_address):
+    """Generates a ptr record from a given IP address.
+
+    Args:
+        ip4_address (str): IPv4 address.
```
Comment (jeffkala) [inline netutils/ip.py:396]: Can we update this?

**Review by jeffkala — CHANGES_REQUESTED** [review]

**Comment by itdependsnetworks** [comment]
I am rethinking this one, I didn't realize that the ptr records already happen with Python ip address, the only one similar is broadcast, and perhaps that was not a great idea to begin with. You can use `netutils.ip.ipaddress_address` method to get the same thing. 

@dgarros what do you think?

File: netutils/ip.py (hunk 2 above)
Comment (qduk) [reply netutils/ip.py:396]: I should've caught that. Thanks Jeff. Updated in case we do use this PR.

**Comment by qduk** [comment]
Closed per comment from @itdependsnetworks

**Pull request closed without merging**